web/src/pages/payment.js

```diff
@@ -0,0 +1,16 @@
+import React, {Component} from 'react';
+
+class Payment extends Component {
+  render() {
+    return (
+      <div>
+        {/* TODO(Myoung-heeSeo): Have to implement payment page. */}
+        {/* TODO(Myoung-heeSeo): Have to delete text */}
+        {/* after develop payment page. */}
```

Comment:
What does this mean?

Reply:
I just mean the text have to delete after implement payment page. I modified comment. Thanks :)